hdf5-types/Cargo.toml

```diff
@@ -14,12 +14,14 @@ categories = ["encoding"]
 
 [features]
 h5-alloc = []
+complex = ["num-complex"]
 
 [dependencies]
 ascii = "1.0"
 libc = "0.2"
 hdf5-sys = { version = "0.8.1", path = "../hdf5-sys" }  # !V
 cfg-if = "1.0.0"
+num-complex = { version = "0.4.0", optional = true, default-features = false }
```

Comment:
@mulimoen Maybe set to `"0.4"`? E.g. because 0.4.2 is already out.

Reply:
Set to 0.4